Helpers/GUIHelper.cs

```diff
@@ -96,4 +97,21 @@ public static string WrapString(string input, int length, DynamicSpriteFont font
 
 		return output;
 	}
+	
+	/// <summary>
+	/// Gets the size of a texture
+	/// </summary>
+	/// <param name="texturePath"></param>
+	/// <returns></returns>
+	public static Vector2? GetSizeOfTexture(string texturePath)

```

Comment:
I would mention that the path has `PathOfTerraria.ModName` prepended to it, as a lot of methods do take full path (i.e. ModName/Path/To/File). This'll save some confusion later.